cmd/minishift/cmd/delete.go

```diff
@@ -98,6 +99,16 @@ func removeInstanceConfigs() {
 	}
 }
 
+func removeProfileFromConfig() {
+	name := profileActions.GetProfileName()
```

Comment:
Unless one does a full remove ,either via `minishift delete --all` or `minishift profile delete [name]`, we should not delete the profile entry.

Reply:
@hferentschik agree.

Reply:
@hferentschik I think we need to do both. I guess users would expect `profile delete` and `minishift delete` should also work for backward compatibility so `minishift delete -all`.